lms/views/lti/basic_launch.py

```diff
@@ -152,8 +156,41 @@ def _show_document(self, assignment):
             assignment, groupings=[self.course]
         )
 
+        if self.request.product.use_toolbar_editing:
+            self.context.js_config.enable_toolbar_editing()
+
+        if self.request.product.use_toolbar_grading and assignment.is_gradable:
```

Comment:
This could be moved to another service or maybe just a private method her.

The main goal of this PR is to have all the grading bar bits together to make a change. Once that other change lands I'll move everything in one go.